kaldi/install.bash

```diff
@@ -14,10 +14,14 @@ prev="-1"
 # TODO: Change this to tue-install-targets --now $name_of_targets
 tue-install-system-now zlib1g-dev automake autoconf patch grep \
     bzip2 gzip wget sox libtool subversion gawk python python3 libatlas3-base \
-    swig zip p7zip-full python-six python-numpy gstreamer1.0-pulseaudio \
+    swig zip p7zip-full python-six python-numpy libglib2.0-dev \
```

Comment:
You are installing numpy here and later with pip. Is that really needed? Looks double to me.